v2/azure-arm.yaml

```
@@ -3758,7 +3753,7 @@ objectModelConfiguration:
         $exportAs: DnsResolversOutboundEndpoint
         $supportedFrom: v2.2.0
       DnsForwardingRuleSet:
-        $export: true
+        $exportAs: DnsForwardingRuleSet
```

The `versions_matrix` file for `api/network` has changed - violating the requirement that no changes are made to the output under `v2/api`. It looks as though this should be 

```suggestion
        $exportAs: DnsForwardingRuleset
```